### DISCUSSION TITLE
[1551530] Pray game 祈愿诗篇

### DISCUSSION THREAD

**Discussion by frankh1111** [pjy612/SteamManifestCache#901] [Share]
### Steam 游戏ID

1551530

### 密钥列表

```hash
1551531;8bf4bb9de5995d70de93ed82c12e20ab74698b9448371dda333a4e7f31e4ad35
1551532;81d1809a02aacdb3d10af04d9637fcea028a71ef0eaf2827af9c07fe12f00998
```

### 清单附件

[1551530.zip](https://github.com/user-attachments/files/17605801/1551530.zip)

### 备注

需要去官网下载补丁，包含无码和重要DLC
[祈愿诗篇 – 补丁](https://kaguragamer.com/product/pray-game-patch/)

**Comment by pjy612** [comment]
* #248
